types/rrdir/index.d.ts

```diff
@@ -3,16 +3,27 @@
 // Definitions by: Zhang Nan <https://github.com/anyone-developer>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+/// <reference lib="esnext.asynciterable" />
```

Comment:
```suggestion
```

This reference comment means that `rrdir` *introduces* `esnext.asynciterable` (even if its not specified by the user), not that it *depends* on it. You already have `/// <reference types="node" />` below, which *does* introduce `esnext.asynciterable` (and already has a lib reference for it), so this line shouldn't be needed.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4486794741ccb9f18f562fefade043ac31e817de/types/node/base.d.ts#L11

Reply:
Ok. Do you have a link handy to some article that might talk about this more in depth? This was one of those pieces I did not understand well.